UX-Guide-Metadata/draft/principles/index.html

```diff
@@ -823,17 +823,80 @@ <h4>Metadata techniques</h4>
-			</div>
+			<p>The guidelines for presenting accessibility metadata detailed in this document are intended to improve the user experience when readers browse the catalogue entry for a publication. However, accessibility metadata also has a vital role to play in helping readers discover publications that are accessible to them.</p>
+			<p>Publication providers, such as vendors and libraries create searching and
+			filtering tools that interpret accessibility metadata to aid in discovery.
+			The set and variety of filters depend on the public they address and
+			the type of book they propose. </p>
+			<p> User feedback stated that in the missing of specific accessibility filters, product details like file format and measures of protections are crucial information.</p>
+			<p>Taking into consideration those realities,
+			the following sections are proposing a minimum and an extension set for filtering options. Mostly any specific information could be added if considered of use for the public of the platform.</p>
+
+
+			<section id="minimum-filtering-set">
+				<h3>Minimum filtering set</h3>
+				<p>
+					Reading systems, commerce, and distribution platforms will typically have specific filtering options; 
+					having uniformity for key aspects and providing guidance for a standardized approach can help 
+					the discovery process for users searching relevant titles. 
+					Accomplishing this, however, should not prevent users with specifics reading needs finding books 
+					they are looking for. To achieve that goal it is recommended that all platforms 
+					present two minimum capabilities, centered around the ways of consuming the content. 
+					These are:
+					<ol>
+						<li>Titles that support non visual reading</li>
+						<li>Titles that support visual adjustments</li>
+					</ol>
+					<p>Of note, only the positive values should be used.</p>
+					<aside class="example" title="Minimum filtering options">
+					<ul>
+						<li>Supports Nonvisual Reading</li>
+						<li>Supports Visual Adjustments</li>
+					</ul>
+				</aside>
+				<div class="note">
+					<p>In 2023, in many markets, we observe very few ebooks with accessibility metadata. 
+					   In such situation, a filter allowing for the discovery of all titles with any accessibility metadata available may be considered with interest. 
+					   We decided not to include that facet in our recommandation because of three factors: 
+					   <li>a risk of finding titles that are informing about unknown accessibility or known limitations;</li>
+					   <li>a limited utility as some accessibility metadata like the presence of TOC will cause the filter to return almost all ebooks;</li>
+					   <li>a technical implementation complexity as it would mean to grab for more than 40 information.</li>
```

Comment:
Not sure what "40 information" is.

Reply:
It sounds like this should say something like "it would require providing more than 40 additional filtering options"

Reply:
I wonder do we need to explain why we have not included this filter? This is an important discussion that we had in our meeting as well as on GitHub issue. We should focus on decision here i.e. two metadata are included in the minimum set. For the reasons why something is excluded, people can visit the GitHub issue.